tests/integration/blockchain/test_exact_economics_model.py

```diff
@@ -87,17 +87,19 @@ def test_exact_economics():
         assert int(365 ** 2 * reward_saturation * decay_half_life / log(2) / (1-multiplier) / expected_lock_duration_coefficient_2) == \
             expected_phase2_coefficient
 
+
+
     # After sanity checking, assemble expected test deployment parameters
     expected_deployment_parameters = (24,       # Hours in single period at genesis
-                                      24,       # Hours in single period
-                                      1053,     # Coefficient which modifies the rate at which the maximum issuance decays (d)
-                                      365,      # Numerator of the locking duration coefficient (k1)
-                                      730,      # Denominator of the locking duration coefficient (k2)
-                                      365,      # Max periods that will be additionally rewarded (awarded_periods)
-                                      2829579800000000000000000000,  # Total supply for the first phase
-                                      1002509479452054794520547,     # Max possible reward for one period for all stakers in the first phase
-                                      30,       # Min amount of periods during which tokens can be locked
-                                      15000000000000000000000,       # min locked NuNits
+                                      24 * 7,   # Hours in single period
+                                      150,     # Coefficient which modifies the rate at which the maximum issuance decays (d)
+                                      52,      # Numerator of the locking duration coefficient (k1)
+                                      52 * 2,  # Denominator of the locking duration coefficient (k2)
+                                      52,      # Max periods that will be additionally rewarded (awarded_periods)
+                                      2829579800000000000000000000,   # Total supply for the first phase
+                                      7036845384615384615384615,      # Max possible reward for one period for all stakers in the first phase
```

Comment:
How's this, and the `d` coefficient, computed?

Reply:
Taken from `BaseEconomics` with new parameters (or just divide by 7)